Sources/Core/AWSClientRuntime/HttpContextBuilder+Extension.swift

```diff
@@ -56,6 +52,16 @@ extension HttpContext {
         // setup client to server
         let messageEncoder = AWSClientRuntime.AWSEventStream.AWSMessageEncoder()
         let messageSigner = AWSClientRuntime.AWSEventStream.AWSMessageSigner(encoder: messageEncoder) {
```

Comment:
have this equal the result of `return signer` and then move this to the very end of the function. The thing being returned should be last to make it clear what is being returned and of what type.

So the end of this whole function should have something like:

```
return try AWSClientRuntime.AWSEventStream.AWSMessageSigner(
        encoder: encoder,
        getSigner: { signer },
        signingConfig: { try await self.makeEventStreamSigningConfig() },
        requestSignature: { self.getRequestSignature() }
    )
}
```

Apply to any other relevant areas of this PR

Reply:
This function doesn't return anything, just initializes things (`messageSigner` and `messageEncoder`) and saves them to middleware context for them to be used by operation stack later.